src/main.rs

```diff
@@ -0,0 +1,96 @@
+use std::ffi::CString;
+use yaca::{self, Key, KeyType, KeyLength, KeyFormat, KeyFileFormat, Private, EncryptAlgorithm, BlockCipherMode, EncryptContext, DecryptContext};
+
+use yaca::{prelude::*, Padding};
+
+
+pub const MSG: &[u8] = b"Lorem ipsum dolor sit amet, consectetuer
+adipiscing elit. Donec hendrerit tempor tellus. Donec pretium posuere
+tellus. Proin quam nisl, tincidunt et, mattis eget, convallis nec,
+purus. Cum sociis natoque penatibus et magnis dis parturient montes,
+nascetur ridiculus mus. Nulla posuere. Donec vitae dolor. Nullam
+tristique diam non turpis. Cras placerat accumsan nulla. Nullam
+rutrum. Nam vestibulum accumsan nisl.";
+
+
+fn main() -> Result<(), Box<dyn std::error::Error>>
+{
+    // Start
+
+    yaca::initialize()?;
+
+    // Keys
+
+    let key = Key::generate(&KeyType::Rsa(Private),
+                            &KeyLength::Bits(512))?;
+    let p = CString::new("dupa")?;
```

Comment:
🤭

Reply:
Let's retain this, it's a decent string.

Reply:
Ok :-)